fastestimator/pipeline/pipeline.py

```diff
@@ -99,6 +100,8 @@ def prepare(self, distribute_strategy=None):
         for mode in self.mode_list:
             self._get_feature_name(mode)
             self._extract_dataset(mode)
+            if mode != "train":
+                distribute_strategy = None
```

Comment:
as in earlier comment, can we not distribute the eval mode?

Reply:
fixed